src/app/containers/FrontPageSection/index.jsx

```diff
@@ -57,11 +97,13 @@ const FrontPageSection = ({ bar, group, sectionNumber }) => {
         visuallyHidden={isFirstSection}
         service={service}
         dir={dir}
+        linkText={isLink ? seeAll : null}
+        href={href}
       >
         {group.strapline.name}
       </SectionLabel>
       {items.length > 1 ? (
-        <StoryPromoUl>
+        <MarginWrapper addMargin={!isFirstSection} as={StoryPromoUl}>
```

Comment:
Since both `TopMargin` and `TopMarginSixHundred`, add margin to the top, what about calling the prop `firstSection` instead of `addMargin`? And then in `MarginWrapper` do `if (firstSection) { ... }`

Reply:
imo this way the props are more descriptive - the MarginWrapper just wraps a component with a `margin-top` - it has no idea what a section or story promo or story promo list is, it just wraps the children its passed. The only logic of whether or not to add the margin is contained in setting the prop `addMargin={!isFirstSection}`, ie. the logic is owned by the FrontPageSection

Reply:
Ok, you convinced me :)